x-pack/plugins/apm/public/components/app/service_node_overview/index.tsx

```diff
@@ -145,25 +145,19 @@ function ServiceNodeOverview({ serviceName }: ServiceNodeOverviewProps) {
   return (
     <>
       <SearchBar />
-      <EuiPage>
-        <EuiFlexGroup direction="column" gutterSize="s">
-          <EuiPanel>
-            <ManagedTable
-              noItemsMessage={i18n.translate(
-                'xpack.apm.jvmsTable.noJvmsLabel',
-                {
-                  defaultMessage: 'No JVMs were found',
-                }
-              )}
-              items={items}
-              columns={columns}
-              initialPageSize={INITIAL_PAGE_SIZE}
-              initialSortField={INITIAL_SORT_FIELD}
-              initialSortDirection={INITIAL_SORT_DIRECTION}
-            />
-          </EuiPanel>
-        </EuiFlexGroup>
-      </EuiPage>
+
+      <EuiPanel hasShadow={false}>
```

Comment:
For standalone `EuiPanel`'s we use `hasShadow={false}`. When there are multiple panels on a single view we still want the shadow.